origin-growth/src/scripts/verifyEvents.js

```diff
@@ -38,7 +43,7 @@ class VerifyEvents {
 
     for (const event of events) {
       let status
-      if (this._isFraud(event)) {
+      if (await this.fraudEngine.isFraudEvent(event)) {
```

Comment:
This can be done in parallel with
```
Promise.all(events.map(event => this.fraudEngine.isFraudEvent(event))) 
...
```

Reply:
Of course the order of promise execution is random here, in case that is important, above solution is no good.

Reply:
Yes, good point that could get parallelized... Though performance is not yet a concern since the volume of events will be low to start with. So if you ok with that I'd prefer processing in order since that could be a useful property at some point later when running the fraud detection heuristics.

Reply:
yup sure 👍